nat/src/main/java/org/hyperledger/besu/nat/NatService.java

```diff
@@ -170,27 +169,16 @@ public static Builder builder() {
   }
 
   public static class Builder {
-    private NatMethod natMethod = NatMethod.NONE;
+    private Optional<NatManager> natManager = Optional.empty();
 
-    public Builder natMethod(final NatMethod natMethod) {
-      checkNotNull(natMethod);
-      this.natMethod = natMethod;
+    public Builder natManager(final Optional<NatManager> natManager) {
+      checkNotNull(natManager);
+      this.natManager = natManager;
       return this;
     }
 
     public NatService build() {
-      return new NatService(buildNatManager(natMethod));
-    }
-
-    /** Initialize the current NatManager. */
-    private Optional<NatManager> buildNatManager(final NatMethod givenNatMethod) {
-      switch (givenNatMethod) {
-        case UPNP:
-          return Optional.of(new UpnpNatManager());
-        case NONE:
-        default:
-          return Optional.empty();
-      }
+      return new NatService(natManager);
```

Comment:
Is a builder required if there's only one parameter?

Reply:
yes you're right I'm going to delete the builder